openshift/mattermost-push-proxy.app.yaml

```diff
@@ -81,17 +71,9 @@ objects:
                 secretKeyRef:
                   name: mattermost-push-proxy-config
                   key: applepushrnpass
-          volumeMounts:
-            - name: config
-              mountPath: /opt/mattermost-push-proxy/config
-        volumes:
-        - name: config
-          persistentVolumeClaim:
-            claimName: mattermost-push-proxy-config
         resources: {}
         terminationMessagePath: /dev/termination-log
         readinessProbe:
-            failureThreshold: 3
```

Comment:
Why was this removed in this PR?

Reply:
that is a typo which I want to correct
but which will work with `oc create` since both values are same and not affecting anything

Reply:
Please confirm that you wanted to delete the line, as a correction to the value. You mentioned `both values`, which values are you talking about?